src/main.py

```diff
@@ -1,14 +1,21 @@
-import parse_fasta
-import sam_to_fasta_file
+from pathlib import Path
+
+from src import parse_fasta
```

Comment:
After installing the package, we can now do this - not only from here, but from any other location on the file system.